bin/ember-template-lint.js

```diff
@@ -166,7 +166,13 @@ function run() {
 
     if (printPending) {
       let failingRules = Array.from(
-        fileErrors.reduce((memo, error) => memo.add(error.rule), new Set())
+        fileErrors.reduce((memo, error) => {
+          if (error.rule) {
```

Comment:
Instead of making this kindof implicit, can we add a special `.code` / `.rule` value that we can check for?

I'm thinking we can tweak this:

https://github.com/ember-template-lint/ember-template-lint/blob/549775f2bc09eb9b7822ed0272ac67460515bc7f/lib/index.js#L174-L178

And add a `rule: 'invalid-pending-module'` or something.

What do you think?

Reply:
Great suggestion! Should be more clear now.